homepage/homepage/content/docs/upgrade/0-20-0.mdx

```diff
@@ -0,0 +1,346 @@
+
+2) **Remove QuickCrypto dependencies/config** (only if you previously added them for `RNQuickCrypto`):
+
+- Remove `react-native-quick-crypto` (and any associated config you added for it).
+- Remove any `SODIUM_ENABLED` / `sodiumEnabled` settings you added specifically for QuickCrypto.
+
+3) **Recommended: remove `CryptoProvider` entirely (RNCrypto is now the default)**
+
+After installing `cojson-core-rn`, you can just rely on the default:
+
+<CodeGroup>
+```tsx
+import { JazzReactNativeProvider } from "jazz-tools/react-native";
+import { MyAppAccount } from "./schema";
+
+const apiKey = "you@example.com";
+
+export function MyJazzProvider({ children }: { children: React.ReactNode }) {
+  return (
+    <JazzReactNativeProvider
+      sync={{ peer: `wss://cloud.jazz.tools/?key=${apiKey}` }}
+      AccountSchema={MyAppAccount}
+    >
+      {children}
+    </JazzReactNativeProvider>
+  );
+}
+```
+</CodeGroup>
+
+If you previously had `CryptoProvider={...}` and/or `import { RNCrypto } ...`, you can remove both:
+
+<CodeGroup>
+```tsx
+// BEFORE
+import { JazzReactNativeProvider } from "jazz-tools/react-native";
+import { RNCrypto } from "jazz-tools/react-native-core/crypto";
+
+export function MyJazzProvider({ children }: { children: React.ReactNode }) {
+  return (
+    <JazzReactNativeProvider
+      sync={{ peer: `wss://cloud.jazz.tools/?key=${apiKey}` }}
+      CryptoProvider={RNCrypto}
+    >
+      {children}
+    </JazzReactNativeProvider>
+  );
+}
+
+// AFTER (recommended: RNCrypto is the default)
+import { JazzReactNativeProvider } from "jazz-tools/react-native";
+
+export function MyJazzProvider({ children }: { children: React.ReactNode }) {
+  return (
+    <JazzReactNativeProvider sync={{ peer: `wss://cloud.jazz.tools/?key=${apiKey}` }}>
+      {children}
+    </JazzReactNativeProvider>
+  );
+}
+```
+</CodeGroup>
+
+4) **If you explicitly used `RNQuickCrypto`, migrate to `RNCrypto`**
+
+If you were using `RNQuickCrypto`:
+
+<CodeGroup>
+```tsx
+// BEFORE
+import { JazzReactNativeProvider } from "jazz-tools/react-native";
+import { RNQuickCrypto } from "jazz-tools/react-native-core/crypto";
+
+export function MyJazzProvider({ children }: { children: React.ReactNode }) {
+  return (
+    <JazzReactNativeProvider
+      sync={{ peer: "wss://cloud.jazz.tools/?key=your-api-key" }}
+      CryptoProvider={RNQuickCrypto}
+    >
+      {children}
+    </JazzReactNativeProvider>
+  );
+}
+
+// AFTER (option A: explicit)
+import { JazzReactNativeProvider } from "jazz-tools/react-native";
+import { RNCrypto } from "jazz-tools/react-native-core/crypto";
+
+export function MyJazzProvider({ children }: { children: React.ReactNode }) {
+  return (
+    <JazzReactNativeProvider
+      sync={{ peer: "wss://cloud.jazz.tools/?key=your-api-key" }}
+      CryptoProvider={RNCrypto}
+    >
+      {children}
+    </JazzReactNativeProvider>
+  );
+}
```

Comment:
This middle step is not required.

We could also merge 4 with 3, as the breaking change is that we don't accept a CryptoProvider anymore.